users/ninjonas/ninjonas.h

```diff
@@ -62,13 +64,13 @@
 
 #define _____________________DVORAK_L1______________________ T_TAB, KC_QUOT, KC_COMM, KC_DOT, KC_P, KC_Y
 #define _____________________DVORAK_L2______________________ T_ESC, KC_A, KC_O, KC_E, KC_U, KC_I
-#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, T_Q, KC_J, KC_K, KC_X
+#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, KC_W, KC_J, KC_K, KC_X
```

Comment:
```suggestion
#define _____________________DVORAK_L3______________________ KC_LSFT, KC_SCLN, KC_Q, KC_J, KC_K, KC_X
```

Reply:
Doh! That explains a lot! Been trying to learn these layouts. Good catch, thank you!